lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1509,15 +1491,15 @@ def _handle_alert_rules(self, relation):
         if not self._charm.unit.is_leader():
             return
 
-        logger.warning("Handling alert rules")
+        logger.debug("Handling alert rules")
```

Comment:
Get rid of this log _entirely_. We have no need to trace in `juju debug-log` at this level.